.github/dependabot.yml

```
@@ -15,9 +17,11 @@ updates:
   directory: /src/pcap
   schedule:
     interval: weekly
```

If we now limit this to one open PR, we should decrease the interval:
```suggestion
    interval: daily
```
Otherwise we have to wait a full week for the next PR and risk piling up dependency updates.

Applies to the go updates below as well.

---

applied